guides/common/modules/proc_installing-a-puppet-module.adoc

```diff
@@ -4,25 +4,21 @@
 = Installing a Puppet module on {Project} server
 
 You can install a pre-built Puppet module from the Puppet Forge.
+Alternatively, you can install a private Puppet module that is not available on the Puppet Forge by manually coping the module to your {ProjectServer}.
+
 The Puppet Forge is a repository that provides Puppet modules contributed by the community.
 Puppet modules flagged as _supported_ are officially supported and tested by Puppet Inc.
 
-[NOTE]
-====
-An alternative way to install a Puppet module is to copy a folder containing the Puppet module to the module path as mentioned below.
-Ensure to resolve its dependencies manually.
-====
-
-This example shows how to add the _ntp module_ to hosts.
-
 .Procedure
-. Navigate to https://forge.puppet.com/[forge.puppet.com] and search for `ntp`.
-One of the first modules is _puppetlabs/ntp_.
-. Connect to your {ProjectServer} using SSH and install the Puppet module:
+. Navigate to https://forge.puppet.com/[forge.puppet.com] and search for the Puppet module you want.
+. Choose how to install the Puppet module:
+* To install from the Puppet forge:
+.. Connect to your {ProjectServer} by using SSH.
+.. Install the module:
```

Comment:
```suggestion
.. On your {ProjectServer}, install the Puppet module:
```

suggestion (non-blocking): we could omit the "use SSH to connect to Foreman". We often start items with "On Foreman Server/On Smart Proxy Server/On your host". This would shorten the procedure.